extensions/migadu/src/aliases.tsx

```diff
@@ -24,7 +24,7 @@ export default function Aliases() {
 
 function AliasesIndex({ domain }: { domain: string }) {
   const { push } = useNavigation();
-  const [aliases, setAliases] = useState<Alias[]>([]);
+  const [aliases, setAliases] = useCachedState<Alias[]>("aliases", []);
```

Comment:
Shouldn't we use `domain` as namespace to avoid cache collisions:

```suggestion
  const [aliases, setAliases] = useCachedState<Alias[]>("aliases", [], { cacheNamespace: domain });
```